tests/testthat/test-cut.R

```diff
@@ -0,0 +1,177 @@
+context("cut function")
+
+with_mock_crunch({
+  ds <- loadDataset("test ds")
+  gen <- ds$gender
+  test_that("cut returns expected output", {
+    
+##################### EXPECTED OUTPUT ######################################
+    basic_output <- list(name = "new_var", derivation = list(`function` = "case", 
```

Comment:
Could you try to make these a little more readable? The regular `dput` output breaks lines in places that aren't the most natural.